sdk/identity/identity/test/internal/node/onBehalfOfCredential.spec.ts

```diff
@@ -81,7 +81,7 @@ describe("OnBehalfOfCredential", function () {
     assert.isNumber(authDetails.result?.expiresOnTimestamp);
   });
 
-  it("authenticates with a certificate assertion", async () => {
+  it.only("authenticates with a certificate assertion", async () => {
```

Comment:
Using this one as the example. To run, use the following command from the identity package directory:

```sh
npx dev-tool run test:vitest -- test/internal/node/onBehalfOfCredential.spec
```